Lib/http/cookiejar.py

```diff
@@ -2004,6 +2004,7 @@ class MozillaCookieJar(FileCookieJar):
     header by default (Mozilla can cope with that).
 
     """
+    httponly_prefix = "#HttpOnly_"
```

Comment:
We haven't visited this module often.  Do you think, we could move this class attributes to a  `__init__` method - The new httponly_prefix as well as the previous `magic_re` and `header`.

Reply:
Just to clarify, you are asking for the class variables to be removed and the values moved into the constructor as literals?

Reply:
Yes.

Reply:
>  Do you think, we could move this class attributes to a ` __init__` method - The new httponly_prefix as well as the previous magic_re and header.

Hi Jacob, 

My original thinking was - we add a  `__init__` method and we move those variables to the `__init__` method, then our usage with `self.attr_name` becomes consistent with instance variable usages.  

Move them to module-level headers is fine too.
 
I notice that moving.compile` to the module header will add a slight performance to the module which was not present. I am fine with it, but I will try to measure the performance of the test suite, before and after to ensure there is no drastic difference with the change.